File: SCALA/Cap05/c05ex09.scala
// c05ex09.scala

import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    var I: Int = 1
    var N, R: Int = 0
    var saida: Boolean = true

    do {
      print("Entre um valor numerico: ")
      N = readInt()

      R = N * 3

      println("Resultado = " + R + "\n")

      if (I > 4) {saida = false}
      I += 1
    } while (saida == true)

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}

File: KOTLIN/Cap05/c05ex09.kt
// c05ex09.kt

fun main() {
    var I: Int = 1
    var N: Int
    var R: Int

    do {
        print("Entre um valor numerico: ")
        N = readLine()?.toInt() ?: 0

        R = N * 3

        println("Resultado = $R\n")

        if (I > 4) break
        I++
    } while (true)

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
